src/pages/models/OMDB_ADDMODEL_DUMMY.tsx

```diff
@@ -0,0 +1,40 @@
+import { GetStaticProps } from 'next';
+import { ParsedUrlQuery } from 'querystring';
+import { useEffect, useState } from 'react';
+import { Model, ModelId } from '../../lib/schema';
+
+import ModelsPage from './[id]';
+
+type Params = ParsedUrlQuery;
+interface Props {
+    modelId: ModelId;
+}
+
+export default function Page({ modelId }: Props) {
```

Comment:
Why do you even have this `modelId` parameter when it's a constant?

Reply:
because nextjs really doesnt like when you have empty props

Reply:
(try it, it doesn't work, and removing getStaticProps breaks it even more)

Reply:
I can't test this right now, because the page doesn't work in general (see comment below), but the following should work properly:


```tsx
import { GetStaticProps } from 'next';
import { useEffect, useState } from 'react';
import { Model, ModelId } from '../../lib/schema';
import ModelsPage from './[id]';

const modelId = 'OMDB_ADDMODEL_DUMMY' as ModelId;

export default function Page() {
    const [model, setModel] = useState<Model | null>(null);

    useEffect(() => {
        const model = JSON.parse(sessionStorage.getItem('dummy-model') ?? '{}') as Model;
        setModel(model);
    }, []);

    if (!model) return null;

    return (
        <div>
            <ModelsPage
                editModeOverride
                modelData={{ [modelId]: model }}
                modelId={modelId}
                similar={[]}
            />
        </div>
    );
}

export const getStaticProps: GetStaticProps = () => {
    return { props: {} };
};

```

Reply:
That worked.